src/python/bot/fuzzers/libFuzzer/engine.py

```diff
@@ -171,8 +172,7 @@ def _merge_new_units(self, target_path, corpus_dir, new_corpus_dir,
     # anyway.
     merge_corpus = self._create_merge_corpus_dir()
 
-    merge_dirs = [new_corpus_dir]
-    merge_dirs.extend(fuzz_corpus_dirs)
+    merge_dirs = copy.copy(fuzz_corpus_dirs)
```

Comment:
nit: `fuzz_corpus_dirs` is a list right?
`copy.copy` isn't needed to copy it. Just do `merge_dirs = fuzz_corpus_dirs[:]`

Reply:
yes, done, thanks!